### PR TITLE
added sample gorillamux-redis app to quickstart

### PR DESCRIPTION
**Pull request by developer-diganta**
# Pull Request Template

## Description

I have added the sample gorilla/mux-redis app to go's quickstart guide.

## Additional Context (Please include any Screenshots/gifs if relevant)

image

## Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [x] I have made corresponding changes to the documentation.
- [ ] I have added corresponding tests.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Comment by SanskritiHarmukh** [comment]
@developer-diganta , Can you make a change at echo-sql sample app page too?

Screenshot from 2023-02-25 01-42-56


- I noticed that the url used for querying is written wrong. Can you correct that to `http://localhost:8082/Lhr4BWAi` ?
- Also, the line below it has a spelling mistake, correct `folderj` to `folder`.

**Comment by SanskritiHarmukh** [comment]
@developer-diganta , Can you make a change at echo-sql sample app page too?

Screenshot from 2023-02-25 01-42-56


- I noticed that the url used for querying is written wrong. Can you correct that to `http://localhost:8082/Lhr4BWAi` ?
- Also, the line below it has a spelling mistake, correct `folderj` to `folder`.

File: docs/go/quickstart/quickstart.md [hunk 1]
```diff
@@ -14,3 +14,4 @@ tags:
 - [URL Shortener with Echo-SQL](/docs/go/quickstart/echo-sql)
 - [URL Shortener with Gin-Mongo](/docs/go/quickstart/gin-mongo)
 - [User Profile with Gin-Mongo](/docs/go/quickstart/gin-mongo-2)
+- [Contact Keeper with Gorilla/Mux-Redis](/docs/go/quickstart/gorillamux-redis)
```
Comment (SanskritiHarmukh) [inline docs/go/quickstart/quickstart.md:17]: Hi @developer-diganta Can you also add a one liner description here about the app?

**Comment by nehagup** [comment]
@developer-diganta are you still working on this?

**Comment by developer-diganta** [comment]
Made a mistake. Will make a new PR 😅

**Pull request closed without merging**